### PR TITLE
feat : Added Cart page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Static Cart page in trendy-t E-Commerce.
Screenshot (287)

### CODE REVIEW

File: src/Pages/Cart-Page/cart.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  padding: 0.5rem;
+  color: var(--white-color);
+}
+.card-product__content--title,
+.card-product__content--price {
+  font-size: var(--font-size-regular);
+}
+.card-product__content--price-line {
+  font-size: var(--font-size-small);
+  color: grey;
```
Comment (SuyashTiwariCoder) [inline src/Pages/Cart-Page/cart.css:99]: You can use variable for color

**Review by SuyashTiwariCoder — APPROVED** [review]
Code looks fine , I have suggested a very minor change and you are good to go .

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

File: src/Pages/Cart-Page/cart.css (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Cart-Page/cart.css:99]: Okay, i will Change Thankyou

**Comment by SaurabhChawda** [comment]
> good to merge

Thankyou

**Pull request merged**